generators/server/templates/src/main/java/package/web/rest/_AccountResource.java

```diff
@@ -408,8 +408,9 @@ public void changePassword(@RequestBody String password) {
     @Timed
     public List<PersistentToken> getCurrentSessions() {
         return persistentTokenRepository.findByUser(
-            userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin())
-                .orElseThrow(() -> new InternalServerErrorException("User could not be found"))
+            userRepository.findOneByLogin(SecurityUtils.getCurrentUserLogin()
```

Comment:
Also here ...

Reply:
Here we would lose some information on the error